src/exponentiation/exponentiation.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+/**
+ * Implement exponentiation function:
```

Comment:
I believe they use exponent instead of exponentiation in English. In this case, `power` is probably even more suitable.

```suggestion
 * Implement exponent function:
```